### ISSUE TITLE
Create Contributions guide

### ISSUE THREAD

**Issue by BrainlessLabs** [BrainlessLabs/momo#6]

**Comment by Sweta271097** [comment]
@BrainlessLabs @sp-mishra Can I take up this issue?

**Comment by Sweta271097** [comment]
@sp-mishra For the contributions guide do we have to create another contributions.md file and add the entire process of how to find an issue, to forking a repository to finally making a PR?

**Comment by sp-mishra** [comment]
Yes we need to. I have added some guide but you can review and then modify it.
